docs/v0.7/en/greptimecloud/usage-&-billing/byoc.md

```diff
@@ -0,0 +1,7 @@
+# BYOC
+
+BYOC (Bring Your Own Cloud) is a service enabling you to utilize your own cloud resources to host GreptimeDB.
```

Comment:
BTW, "host GreptimeDB" sounds a bit confusing. The cloud provision is far more than GreptimeDB. But it may be intuitive for users. WDYT @sunng87?

Reply:
I think it's ok to use *host* here, especially for BYOC plan. We consider the cloud instance is an enhanced version of greptimedb.